### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by Olga-koml**

### CODE REVIEW

File: file_1.py [hunk 1]
```diff
@@ -1 +1,19 @@
...
+game_over=False #Создаём переменную, которая поможет нам контролировать 
+статус игры — завершена она или нет. Изначально присваиваем значение False,
+ то есть игра продолжается.
+ 
+while not game_over:
+   for event in pygame.event.get():
+       print(event)  #Выводить в терминал все произошедшие события.
+ 
+pygame.quit()
+quit()
```
Comment (RoutTufuch) [inline file_1.py:19]: Well done!!!! :)

**Review by RoutTufuch — COMMENTED** [review]
Hello, Olga!
Just finished my first review! I guess, that it was not too much informative for you, but really useful for me!

File: file_1.py (hunk 1 above)
Comment (Olga-koml) [reply file_1.py:19]: thanks a lot

**Review by RoutTufuch — CHANGES_REQUESTED** [review]
Please, the only thing which we need to change - is to adjust the name of your virtual environment! Use venv or env!

**Review by RoutTufuch — APPROVED** [review]
Okay, so, maybe we can continue with vevn) For fun!

**Pull request merged**